<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainTabWidget</class>
    <widget class="QWidget" name="MainTabWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>cocr app</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QTabWidget" name="tabWidget">
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <property name="tabPosition">
                        <enum>QTabWidget::South</enum>
                    </property>
                    <property name="tabShape">
                        <enum>QTabWidget::Rounded</enum>
                    </property>
                    <property name="currentIndex">
                        <number>1</number>
                    </property>
                    <property name="iconSize">
                        <size>
                            <width>32</width>
                            <height>32</height>
                        </size>
                    </property>
                    <property name="elideMode">
                        <enum>Qt::ElideNone</enum>
                    </property>
                    <property name="documentMode">
                        <bool>false</bool>
                    </property>
                    <property name="tabsClosable">
                        <bool>false</bool>
                    </property>
                    <property name="movable">
                        <bool>false</bool>
                    </property>
                    <property name="tabBarAutoHide">
                        <bool>false</bool>
                    </property>
                    <widget class="QWidget" name="wel_tab">
                        <property name="sizePolicy">
                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                <horstretch>0</horstretch>
                                <verstretch>0</verstretch>
                            </sizepolicy>
                        </property>
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/home.png</normaloff>:/images/home.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>home</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="draw_tab">
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/draw.png</normaloff>:/images/draw.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>draw</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="v2d_tab">
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/ben2d.png</normaloff>:/images/ben2d.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>view2d</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="v3d_tab">
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/ben3d.png</normaloff>:/images/ben3d.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>view3d</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="img_tab">
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/filesave.png</normaloff>:/images/filesave.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>file</string>
                        </attribute>
                    </widget>
                    <widget class="QWidget" name="cam_tab">
                        <attribute name="icon">
                            <iconset resource="../../../../assets/COCR_res.qrc">
                                <normaloff>:/images/shutter.png</normaloff>:/images/shutter.png
                            </iconset>
                        </attribute>
                        <attribute name="title">
                            <string>camera</string>
                        </attribute>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../../../../assets/COCR_res.qrc"/>
    </resources>
    <connections/>
</ui>
